sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/DoFn.java

```diff
@@ -214,6 +214,30 @@
     protected abstract <AggInputT, AggOutputT>
         Aggregator<AggInputT, AggOutputT> createAggregator(
             String name, CombineFn<AggInputT, ?, AggOutputT> combiner);
+
+    /**
+     * Sets up {@link Aggregator}s created by the {@link DoFn} so they are usable within this context.
+     *
+     * <p>This method should be called by runners before the {@link StartBundle @StartBundle} method.
+     */
+    @Experimental(Kind.AGGREGATOR)
+    protected final void setupDelegateAggregators() {
```

Comment:
This is copy/paste. Previously, aggregators all resided on the `OldDoFn` wrapper. I decided this approach was fine since both the old copy AND aggregators are likely to go away soon.